packages/api-demo/src/app.json

```diff
@@ -4,7 +4,8 @@
         "name": "SSF Desktop Wrapper",
         "url": "http://localhost:8080/index.html",
         "uuid": "ssf-desktop-api-openfin-demo",
-        "autoShow": true
+        "autoShow": true,
+        "preload": "http://localhost:8080/scripts/containerjs-api-bundle.js"
```

Comment:
And another question! Does this `preload` property ensure that this scripts is preloaded for any window opened via `new fin.desktop.Window()`?